site/public/js/grade-inquiry.js

```diff
@@ -1,5 +1,25 @@
 /* global buildCourseUrl, WebSocketClient */
-/* exported initGradingInquirySocketClient, onComponentTabClicked, onGradeInquirySubmitClicked, onReady, onReplyTextAreaKeyUp */
+/* exported saveDraft, initGradingInquirySocketClient, onComponentTabClicked, onGradeInquirySubmitClicked, onReady, onReplyTextAreaKeyUp */
+
+function saveDraft() {
+    //saveDraft function displays the saved draft text on grade inquiry box
+    let draftContent = localStorage.getItem('content');
+    if (draftContent === null) {
+        draftContent = {};
+    }
+    else {
+        draftContent = JSON.parse(draftContent);
+    }
+    const elements = document.getElementsByClassName('markdown-textarea fill-available ');
```

Comment:
It would be better to identify elements via a more specific identifier, such as by ID, or by class.  Additionally, it would be better to use jQuery for this.

Reply:
![image](https://github.com/Submitty/Submitty/assets/122962727/6ece9cd9-8b1b-4e44-b79e-92ecf378d608)
It doesn't seem to have a more specific identifier we can use, I have also switched into jquery.